### ISSUE TITLE
Document the various environment variables used, including the new OPENBLAS_DEFAULT_NUM_THREADS

### ISSUE THREAD

**Issue by mattip** [xianyi/OpenBLAS#3967]
I was intrigued by the mention of `OPENBLAS_DEFAULT_NUM_THREADS` and looked around for what it does. My goto place for this is the FAQ page in the wiki, but I could not find it there. In addition to documenting the new `OPENBLAS_DEFAULT_NUM_THREADS`, it would be nice to have a table somewhere of all the environment variables used in OpenBLAS and their uses.

**Comment by martin-frbg** [comment]
Right, this could probably go into the USAGE.md document ? (Though the "traditional" environment variables related to the runtime thread limit are mentioned in README.md). And maybe the case for OPENBLAS_DEFAULT_NUM_THREADS needs revisiting, as there have been some fixes to the thread limits code in the meantime. The period between releases has just grown too much lately, I'm hoping to get back to the earlier 2/3 months cycle)
